lib/shared/permissions.js

```diff
@@ -143,14 +158,18 @@ exports.getEnvironmentCommandPrefix = (environment) => {
  * });
  */
 exports.elevateCommand = (command, options) => {
-  const prefixedCommand = _.concat(exports.getEnvironmentCommandPrefix(options.environment), command);
+  const prefixedCommand = _.concat(
+    exports.getEnvironmentCommandPrefix(options.environment),
+    _.map(command, quoteString)
+  );
 
   if (os.platform() === 'win32') {
     const elevator = Bluebird.promisifyAll(nativeModule.load('elevator'));
-    return elevator.elevateAsync(_.concat([
+    return elevator.elevateAsync([
       'cmd.exe',
-      '/c'
-    ], prefixedCommand)).then((results) => {
+      '/c',
+      `"${_.join(prefixedCommand, ' ')}"`
```

Comment:
In an earlier version of this PR, I suggested this could be using `quoteString` too? I can't see why not to do that here?